### DISCUSSION TITLE
How to set name for input hidde

### DISCUSSION THREAD

**Discussion by Truongcot99** [sa-si-dev/virtual-select#117] [Q&A] [answered]
how to set name for input hidde, i tried but it doesn't work.

//Code
VirtualSelect.init({
        ele: 'select.career',
        required: true,
        multiple: true,
        search: true,
        name: 'test',
    });

**Comment by sa-si-dev** [comment, marked as answer]
@Truongcot99 it should work as expected. Please share the `select.career` element's HTML code and what you get in the browser for the hidden input element.

**Reply by Truongcot99** [reply]
Thanks, I did it lol
